fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -151,6 +303,42 @@ private fun SettingsCard(
     }
 }
 
+private fun HighlightType.toHighlight() =
+    when (this) {
+        HighlightType.QUALITY -> Highlight.QUALITY
+        HighlightType.PRICE -> Highlight.PRICE
+        HighlightType.SHIPPING -> Highlight.SHIPPING
+        HighlightType.PACKAGING_AND_APPEARANCE -> Highlight.PACKAGING_AND_APPEARANCE
+        HighlightType.COMPETITIVENESS -> Highlight.COMPETITIVENESS
+    }
+
+// As part of Bug 1841600, update iconResourceId for each highlight type.
+private enum class Highlight(
```

Comment:
Appreciate you updating this 👍 non-blocking question, should we call this something like HighlightUiData?

Reply:
for sure! thanks for the review! 

hmm I think even the `State` objects are ui `data/model/state`. Being a private class for a composable does say that it's a `UIData`, it think maybe we don't need the extra verbosity as it's clear it's a ui data. wdyt?

Reply:
That's a fair point - happy to keep as-is :shipit: